run.py

```diff
@@ -418,7 +418,11 @@ def check_for_update():
 
 def print_out_update():
   import pip
-  SemVer = pip.util.version.SemanticVersion
+  try:
+    SemVer = pip._vendor.distlib.version.SemanticVersion
+  except:
```

Comment:
`import pip` is not party of the `try/except` so I'm guessing it will be the `AttributeError`.. no?